### ISSUE TITLE
Scroll not working on mobile devices, only desktop. 

### ISSUE THREAD

**Issue by AyoParadis** [XzaR90/react-router-hash-link#1]
The scroll only works if you double tap. single tap scrolls to the top of the screen on mobile.

https://css-tricks.com/annoying-mobile-double-tap-link-issue/ this solution did not work

**Comment by AyoCodess** [comment]
The scroll only works if you double tab. single tap scrolls to the top of the screen on mobile.

**Comment by XzaR90** [comment]
Do you think this could be the issue?

https://github.com/XzaR90/react-router-hash-link/blob/aa890b0a946784343b73dcc4207f12c7f628cd62/src/checkIfCanScroll.ts#L15

**Comment by AyoCodess** [comment]
Have you tested it? I'll have a look at it. but if you do before me, please let me know.

**Comment by AyoParadis** [comment]
> checkIfCanScroll.ts

Are you experiencing the same issues on mobile?

It's a strong candidate.

**Comment by XzaR90** [comment]
I will see if I can fix it this weekend. I don't have any good way to test this. I have a Iphone 13 Pro and it seems to be working as it should on it with the current version. What phone did you test with?

**Comment by XzaR90** [comment]
@AyoCodess I don't think it is the checkIfCanScroll that is the issue as react-router is using the same code.
Could you help me with specifying which phone and browser is used?

**Comment by AyoParadis** [comment]
Iphone 11 max. Same issue on all browsers.
